src/main/resources/build.gradle.kts.mustache

```diff
@@ -11,7 +11,9 @@ plugins {
     idea
 
     // spotless
-    id("com.diffplug.gradle.spotless") version "3.26.0"
+    id("com.diffplug.gradle.spotless") version "4.0.1"
+
+    id("com.github.ben-manes.versions") version "0.28.0"
```

Comment:
should more of `build.gradle.kts` be in `build.gradle.kts.mustache'?

Reply:
depends on what goodies you want the generated projects to get by default. less is more? 🤯